src/atlannot/evaluation.py

```diff
@@ -55,13 +55,21 @@ def compute_jaggedness(
         Dictionary containing the region id as keys and the mean of the
         jaggedness of that given region id as values.
     """
-    metrics = core.compute(volume, coronal_axis_index=axis, regions=region_ids)
+    try:
+        metrics = core.compute(volume, coronal_axis_index=axis, regions=region_ids)
+    except Exception:
+        return {}
+
```

Comment:
- Yes, I can try to create a ticket to ask if it is possible to replace by another (more specific) exception. 
- This exception is raised only if none of the region_ids is present.

Maybe we can do something similar (add a parameter for the `precomputed_all_region_ids`) to avoid computing this every time.

Reply:
What do you think about this change (https://github.com/BlueBrain/atlas-annotation/commit/b9b5ed77679fe23d4e6c3cd5ef177d79965b110a) ?